ctsit/project_overlay_banner
# Initial working module

Testing procedures:
1. Enable this module and activate it on any project(s).
2. Alter the Control Center level configuration settings and observe that they apply to all projects.
3. Alter the Project level configuration settings and observe they are applied (if they are allowed in the Control Center configuration toggles).

The banner should appear on _all_ project pages, including surveys.

Configuration settings should be able to override each other pending admin level toggles and if the configuration is blank. If a configuration field is left blank the next level should be used. The order of overrides should be as follows:  
Project settings (if allowed) -> Control Center settings -> Default settings

Known issue: the text alignment built in to the **Banner Text** field does not work as expected.
